### PR TITLE
Cleaned up and updated the CoC

### PR DESCRIPTION
**Pull request by KiriCattus**
Cleaned up the channels section, edited some of the role requirements and added two new roles to the list.

### CODE REVIEW

File: docs/CoC.md [hunk 1]
```diff
@@ -47,31 +49,40 @@ Keep in mind that some members have weird ways of interacting with each other. D
...
  * Manage Channel
+ * Sync Permissions (This will break your channels permissions)
 
 In addition, changing the permissions for Staff, Moderators, or MMD Admins is prohibited, as is changing your channel category. Violations of these rules may lead to having your channel removed and an infraction added to the channel owner or owners.
 
 If a user channel is inactive for 30+ days, a staff member will send a message asking the channel owner to contact staff within 7 days. If they do not contact staff within that time frame the channel will be archived.
 
-### Special Channels
+## Channels
+- **#readme** - This channel has information about MMD and it's rules. Please make sure you read through it.
```
Comment (antoinegag) [inline docs/CoC.md]: "MMD and it's rules" -> "MMD and **its** rules"

File: docs/CoC.md [hunk 2]
```diff
@@ -47,31 +49,40 @@ Keep in mind that some members have weird ways of interacting with each other. D
...
-- **#mmd-public-server** - The chat for the public minecraft server. Also contains a relay between server chat and discord.
-- **#mmd-dev-server** - The chat for the private dev server. Also contains a relay between server and discord chat.  
-- **#modpack-talk** - The place used to ask questions about modpack development, and discussion relating to modpacks.
+- **#mmd-public-server** - The chat for the public Minecraft server. Also contains a relay between server chat and discord.
+- **#mmd-dev-server** - The chat for the whitelisted Minecraft dev server. Also contains a relay between server and discord chat. 
 - **#showcase** - Used to show new content made by the community, not for posting mod announcements.
-- **#streams** - Used by streamers to let others know they're streaming. Usually done through a bot.
+- **#modpacks** - The place used to ask questions about modpack development, and discussion relating to modpacks.
+- **#art** - Used to discuss art you've made, or wanted to share. Keep the chat SFW!
+- **#stopthatitsverysilly** - Shitposting, politics, memes, inflammatory but not racist or hateful stuff.
```
Comment (antoinegag) [inline docs/CoC.md]: And no anime!

File: docs/CoC.md [hunk 3]
```diff
@@ -47,31 +49,40 @@ Keep in mind that some members have weird ways of interacting with each other. D
...
+### DEVELOPMENT
+- **#mod-development** - The place to ask questions about making mods, getting help with your mod code issues.
 - **#requests** - Place for general requests, such as new mods, modpacks, art and so on.
-- **#artist-talk** - A chat room specifically for artists to talk in.
-- **#stopthatitsverysilly** - Shitposting, politics, memes, inflammatory but not racist or hateful stuff. 
+- **#diluv** - The place to find information on Diluv and discuss ideas, features and the service as a whole.
+- **#forgegradle3-development** - ForgeGeadle 3 development progress and discussion. Ask questions and give ideas for FG3 here.
+- **#other-development** - Talk about other none Minecraft related projects you are working on and ask for help with code here.
+### MISC
+- **#role-requests** - Request a modder, artist, translator, modpack maker or streamer(Twitch/YouTube) here. (See Role Requirements for more information)
```
Comment (antoinegag) [inline docs/CoC.md]: I wouldn't specify the platform for the streamer role.

But if you do maybe add Mixer.

File: docs/CoC.md [hunk 4]
```diff
@@ -47,31 +49,40 @@ Keep in mind that some members have weird ways of interacting with each other. D
...
+- **#channel-management** - Request a channel for your mod/s here (You must have the Modder role already to request a channel.
```
Comment (antoinegag) [inline docs/CoC.md]: Missing a closing ) at the end

**Review by antoinegag — CHANGES_REQUESTED** [review]

File: docs/CoC.md (hunk 1 above)
Comment (KiriCattus) [reply docs/CoC.md]: Discord always corrects it to it's rather than its o.O grammar thing I assume?

**Review by antoinegag — APPROVED** [review]

File: docs/CoC.md [hunk 5]
```diff
@@ -47,31 +49,40 @@ Keep in mind that some members have weird ways of interacting with each other. D
...
+### DEVELOPMENT
+- **#mod-development** - The place to ask questions about making mods, getting help with your mod code issues.
 - **#requests** - Place for general requests, such as new mods, modpacks, art and so on.
-- **#artist-talk** - A chat room specifically for artists to talk in.
-- **#stopthatitsverysilly** - Shitposting, politics, memes, inflammatory but not racist or hateful stuff. 
+- **#diluv** - The place to find information on Diluv and discuss ideas, features and the service as a whole.
+- **#forgegradle3-development** - ForgeGeadle 3 development progress and discussion. Ask questions and give ideas for FG3 here.
+- **#other-development** - Talk about other none Minecraft related projects you are working on and ask for help with code here.
+### MISC
+- **#role-requests** - Request a modder, artist, translator, modpack maker or streamer (Twitch, YouTube and Mixer) here. (See Role Requirements for more information)
```
Comment (antoinegag) [inline docs/CoC.md]: Actually I think we should just get rid of the platform here, not really needed and will force us to always update for every new platform that come and goes.

**Review by jriwanek — APPROVED** [review]

File: docs/CoC.md (hunk 5 above)
Comment (KiriCattus) [reply docs/CoC.md]: Either before this is merged or I get home to fix it someone should remove this as I can't remember how I did it on mobile...

**Comment by Dizzy611** [comment]
in the part talking about curseforge, minor spelling error discription->description.

flow/structural change:

 To request one of the roles below, with the exception of Partner, see the requirements and contact a moderator (@Moderators) in the #role-requests channel.

to

 To request one of the roles below you must familiarize yourself with the requirements and, unless otherwise noted, contact a moderator (@Moderators) in the #role-requests channel.


this also allows us to seamlessly add more roles that function similarly to partner in future without changing this line.

**Pull request merged**